Packs/CommonScripts/ReleaseNotes/1_4_31.md

```diff
@@ -0,0 +1,8 @@
+
+#### Scripts
+
+##### LoadJSON
+
+- Fixed an issue where control charachter presented in the input
+- Fixed an issue where the command failed on inputs that contained tabs.
```

Comment:
note the conflicts here

Reply:
@bakatzir It fixed control characters problem, not tabs only tabs